<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-4RBB3DGA" failures="0" tests="3" name="Default test" time="0.469" errors="0" timestamp="25 Jul. 2021 16:51:05 GMT">
  <testcase classname="com.qa.para.base.BaseTest" name="@BeforeTest setUp" time="0.046">
    <failure type="java.lang.NullPointerException" message="key can&amp;apos;t be null">
      <![CDATA[java.lang.NullPointerException: key can't be null
at java.base/java.lang.System.checkKey(System.java:960)
at java.base/java.lang.System.getProperty(System.java:830)
at com.qa.para.base.BasePage.init_prop(BasePage.java:81)
at com.qa.para.base.BaseTest.setUp(BaseTest.java:23)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest setUp -->
  <testcase classname="com.qa.para.base.BaseTest" name="@AfterTest tearDown" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterTest tearDown -->
  <testcase classname="com.qa.para.tests.LoginPageTest" name="verifyForgotPWDLinkTest" time="0.0">
    <skipped/>
  </testcase> <!-- verifyForgotPWDLinkTest -->
  <testcase classname="com.qa.para.tests.LoginPageTest" name="verifyLoginpageTitleTest" time="0.0">
    <skipped/>
  </testcase> <!-- verifyLoginpageTitleTest -->
  <testcase classname="com.qa.para.tests.LoginPageTest" name="loginTest" time="0.0">
    <skipped/>
  </testcase> <!-- loginTest -->
</testsuite> <!-- Default test -->
